iwyu_tool.py

```diff
@@ -506,6 +515,7 @@ def custom_help():
                               'is greater than the provided value'))
     parser.add_argument('-p', metavar='<build-path>', required=True,
                         help='Compilation database path', dest='dbpath')
+    parser.add_argument('-e', '--exclude', action='append', default=[])
```

Comment:
Sorry I missed this earlier, but we need a help text for this arg too.

I can add it before merging unless you beat me to it.